setup.py

```diff
@@ -86,7 +86,7 @@
             "pytz",
             "snowflake-connector-python[pandas,secure-local-storage]>=3.0.2",
             "sqlalchemy-stubs",
-            "tenacity==8.1.0",
+            "tenacity",
```

Comment:
Note: this was pinned in [this commit](https://github.com/TobikoData/sqlmesh/commit/0ab79bbb3e87d4827391a404aa90f7fe27472332) back when we were using Airflow 2.3.3.

Pinning it was causing an ancient `dbt-athena-community` to be resolved. Since we've moved on to newer Airflow in the meantime, i've removed the pin